app/assets/javascripts/discourse/tests/helpers/component-test.js

```diff
@@ -12,6 +12,7 @@ import { currentSettings } from "discourse/tests/helpers/site-settings";
 
 export function setupRenderingTest(hooks) {
   emberSetupRenderingTest(hooks);
+  let _oldAutocomplete;
```

Comment:
Oh my f*cking gawd! That was the part that was _randomly_ breaking the `rake qunit:test` depending on whether a test calling this `setupRenderingTest` method would run **before** any autocomplete tests, since this was basically nullifying the `autocomplete` "plugin".

Saving the method and restoring it once the "component test" is done made it all work 🎉 

(cc @CvX - that was from a [commit](https://github.com/discourse/discourse/commit/f1136481078b6f3e12f6c571134830847e96250d) from 3 years ago 😅)

Reply:
Nothing... So it's gone now 🔥